src/Controller/Backend/ProfileController.php

```diff
@@ -67,7 +67,7 @@ public function profileEditPost(Request $request, UrlGeneratorInterface $urlGene
         ];
 
         // Validate username
-        if (! filter_var(mb_strlen($user->getFullName()), FILTER_VALIDATE_INT, $usernameValidateOptions)) {
+        if (! filter_var(mb_strlen($user->getDisplayName()), FILTER_VALIDATE_INT, $usernameValidateOptions)) {
```

Comment:
Move this to separate method / class (or just add a todo)

Reply:
Actually I see there is a Validator class so let's move it there or to separate UserValidator

Reply:
This one will need more extensive work later, so I don't want to do that just yet to prevent double work.

Reply:
Ok let's add a todo

Reply:
Done